docs/Configuration.md

```diff
@@ -703,6 +703,10 @@ async runTests(
 ): Promise<void>
 ```
 
+If you need to restrict your test-runner to only run in serial rather then being
+exectued in paralell your class should have the static attribute `serial` to be
```

Comment:
can we call it `isSerial`?

Reply:
sure 👍

Reply:
typos: 
- `exectued` => `executed`
- `paralell` => `parallel`